src/sdl2/locale.rs

```diff
@@ -0,0 +1,93 @@
+//! System locale information.
+
+/// A locale defines a user's language and (optionally) region.
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct Locale {
+    pub lang: String,
+    pub country: Option<String>,
+}
+
+impl ToString for Locale {
+    fn to_string(&self) -> String {
+        let mut hint = self.lang.to_string();
+
+        if let Some(region) = &self.country {
+            hint.push('_');
+            hint.push_str(&region);
+        }
+
+        hint
+    }
+}
```

Comment:
Implement `Display` instead of `ToString`, see https://doc.rust-lang.org/std/string/trait.ToString.html